app/database/seeds/data/location_event.json

```diff
@@ -0,0 +1,3 @@
+[
+{"id": "c74757d1-fc04-3006-8982-9839a84612f7", "name": "Music Building Going But Will Not Be Forgotten", "external_web_url": "http://www.uwindsor.ca/dailynews/2018-04-10/music-building-going-not-forgotten", "description": "The University of Windsor's Music Building was demolished.", "when": "2018-04-11"}
```

Comment:
Get a recent dump of the production database and test that importers/utils/json_to_db.py will update this after running 'php artisan migrate' on it.